chain/beacon/node.go

```diff
@@ -358,7 +358,7 @@ func (h *Handler) broadcastNextPartial(current roundInfo, upon *chain.Beacon) {
 	ctx := context.Background()
 	previousSig := upon.Signature
 	round := upon.Round + 1
-	beaconID := h.conf.Group.ID
+	beaconID := commonutils.GetCorrectBeaconID(h.conf.Group.ID)
```

Comment:
Can we name it other than "Correct", something like "Canonical" maybe ? correct is very subjective.